Userland/Libraries/LibJS/Runtime/Temporal/PlainDate.cpp

```diff
@@ -4,12 +4,69 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <LibJS/Runtime/AbstractOperations.h>
+#include <LibJS/Runtime/GlobalObject.h>
 #include <LibJS/Runtime/Temporal/Calendar.h>
 #include <LibJS/Runtime/Temporal/PlainDate.h>
+#include <LibJS/Runtime/Temporal/PlainDateConstructor.h>
+#include <LibJS/Runtime/Temporal/PlainDateTime.h>
 #include <LibJS/Runtime/Value.h>
 
 namespace JS::Temporal {
 
+// 5 Temporal.PlainDateTime Objects, https://tc39.es/proposal-temporal/#sec-temporal-plaindatetime-objects
+PlainDate::PlainDate(i32 year, i32 month, i32 day, Object& calendar, Object& prototype)
+    : Object(prototype)
+    , m_year(year)
+    , m_month(month)
+    , m_day(day)
+    , m_calendar(calendar)
+{
+}
+
+void PlainDate::visit_edges(Visitor& visitor)
+{
+    visitor.visit(&m_calendar);
+}
+
+// 3.5.1 CreateTemporalDate ( isoYear, isoMonth, isoDay, calendar [ , newTarget ] ), https://tc39.es/proposal-temporal/#sec-temporal-createtemporaldate
+PlainDate* create_temporal_date(GlobalObject& global_object, i32 year, i32 month, i32 day, Object& calendar, FunctionObject* new_target)
```

Comment:
Let's use `iso_*` here :)

Reply:
fixed both here and in the PlainDate object itself